api_tests/institutions/views/test_institution_department_list.py

```diff
@@ -136,6 +136,22 @@ def test_get(self, app, url, admin, institution, populate_counts):
             'links': {'self': f'http://localhost:8000/v2/institutions/{institution._id}/metrics/departments/'}
         }]
 
+        # Tests CSV Export
```

Comment:
Can we test the headers are correct? I noticed that the sometimes say `attributes.X` instead of just `X` but the sample doesn't reflect it. Even if that's just  djangorestframework-csv magic always a good idea to explictly confirm excel formatting data if it comes directly from product, so it doesn't get deleted by future devs just trying to remove technically extraneous code.

Reply:
That's a good point - I added an assert to that effect.